if double buffering is enabled, but vbeSetDisplayStart() fails
turn double buffering off.

vbeSetDisplayStart always fails on VMWare.

diff -Nur MPlayer-20060519.orig/libvo/vo_vesa.c MPlayer-20060519/libvo/vo_vesa.c
--- MPlayer-20060519.orig/libvo/vo_vesa.c	Sat Apr 29 02:06:42 2006
+++ MPlayer-20060519/libvo/vo_vesa.c	Fri May 26 19:00:04 2006
@@ -1022,7 +1022,12 @@
 	}
 	if(HAS_DGA() && vo_doublebuffering)
 	{
-	    for(i=0;i<MAX_BUFFERS;i++)
+	    if (VBE_OK != vbeSetDisplayStart(0, vo_vsync))
+	    {
+ 		mp_msg(MSGT_VO,MSGL_WARN, "[VO_VESA] Can't use double buffering: changing displays failed.\n");
+		multi_size = 1;
+	    }
+	    for(i=0;i<multi_size;i++)
 	    {
 		win.ptr = dga_buffer = video_base + multi_buff[i];
                 clear_screen();	/* Clear screen for stupid BIOSes */
